<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProfile</class>
 <widget class="QWidget" name="NewProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>281</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Nákupní Lístek JV</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#Instructions {
color: rgb(255, 255, 255);
font: 24pt &quot;FreeSans&quot;;
}

QWidget#NewProfile {
background-color: rgb(0, 0, 34);
}

QPushButton:hover#Button_Save_Continue{
border: 3px solid rgb(255, 255, 255);
background-color: rgb(0, 0, 53);
}

QPushButton:hover#Button_Cancel{
border: 3px solid rgb(255, 255, 255);
background-color: rgb(0, 0, 53);
}

QLineEdit:hover#Input_name{
border: 3px solid rgb(255, 255, 255);
}

QLabel#Label_error_message{
color: rgb(255, 0, 0);
font: 8pt &quot;FreeSans&quot;;
}

QLabel#Label_Name{
color: rgb(255, 255, 255);
font: 14pt &quot;FreeSans&quot;;
}

QLineEdit#Input_name{
color: rgb(255, 255, 255);
font: 14pt &quot;FreeSans&quot;;
border: 2px solid rgb(255, 255, 255);
background-color: rgb(0, 0, 53);
border-radius: 20px;
padding-left: 20px;
padding-right: 20px;
}

QPushButton#Button_Save_Continue{
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border: 2px solid rgb(255, 255, 255);
border-radius: 20px;
font: 12pt &quot;FreeSans&quot;;
}

QPushButton#Button_Cancel{
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border: 2px solid rgb(255, 255, 255);
border-radius: 20px;
font: 12pt &quot;FreeSans&quot;;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="Label_error_message">
      <property name="geometry">
       <rect>
        <x>169</x>
        <y>60</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Save_Continue">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>360</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Uložit a pokračovat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Cancel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>360</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Zavřít okno</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Input_name">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>280</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="Label_Name">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Jméno:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Instructions">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>151</width>
        <height>39</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Nový profil</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
